layers/+emacs/org/packages.el

```diff
@@ -82,6 +83,11 @@
   (spacemacs|use-package-add-hook mu4e
     :post-config (require 'org-mu4e nil 'noerror)))
 
+(defun org/pre-init-notmuch ()
+  ;; Load org-notmuch when notmuch is actually loaded
+  (spacemacs|use-package-add-hook notmuch
+    :post-config (require 'org-notmuch nil 'noerror)))
+
```

Comment:
1. How will this work when spacemacs doesn't provide a `notmuch` layer and there isn't a `init-notmuch` in spacemacs layers? 

2. If spacemacs isn't providing notmuch - is it right to add this configuration here?